scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/spel/Typer.scala

```diff
@@ -786,7 +787,20 @@ private[spel] class Typer(
       classDefinitionSet,
       evaluationContextPreparer,
       anyMethodExecutionForUnknownAllowed,
-      dynamicPropertyAccessAllowed
+      dynamicPropertyAccessAllowed,
+      absentVariableReferenceAllowed
+    )
+
+  def withAbsentVariableReferenceAllowed(value: Boolean): Typer =
+    new Typer(
+      dictTyper,
+      strictMethodsChecking,
+      staticMethodInvocationsChecking,
+      classDefinitionSet,
+      evaluationContextPreparer,
+      anyMethodExecutionForUnknownAllowed,
+      dynamicPropertyAccessAllowed,
+      value
```

Comment:
use named parameters to make sure that this boolean is passed as a correct parameter